scss/components/_pagination.scss

```diff
@@ -50,10 +50,15 @@ $pagination-item-color-disabled: $medium-gray !default;
 /// @type Color
 $pagination-ellipsis-color: $black !default;
 
-/// If `false`, don't display page number links on mobile, only next/previous links.
+/// If `false`, don't display page number links on mobile, only next/previous links
+/// and optionally current page number.
 /// @type Boolean
 $pagination-mobile-items: false !default;
 
+/// If `false`, don't display current page number on mobile.
+/// @type Boolean
+$pagination-mobile-current-item: false !default;
```

Comment:
Shouldn't this be defaulted to true? This way it's an unnecessery breaking change 😟